GameServerCore/Domain/ICharData.cs

```diff
@@ -1,4 +1,5 @@
-﻿using GameServerCore.Enums;
+﻿using GameServerCore.Domain;
+using GameServerCore.Enums;
 
 namespace LeagueSandbox.GameServer.Content
```

Comment:
I assume that you should change the namespace here to `GameServerCore.Domain` as in other files?